### PR TITLE
Dac6 2166

### PR DESCRIPTION
**Pull request by AashuChauhanHmrc**

### CODE REVIEW

File: app/models/BusinessType.scala [hunk 1]
```diff
@@ -53,6 +56,15 @@ object BusinessType extends Enumerable.Implicits {
     Radios(field, items)
   }
 
+  def options(implicit messages: Messages): Seq[RadioItem] = values.zipWithIndex.map {
```
Comment (naree-hmrc) [inline app/models/BusinessType.scala:59]: I am not comfortable with the view layer code mixed with the models but I can see that this code is consistent with the rest of the projects. I am happy to leave it as it is for now but perhaps we can think about how to achieve better separation.

**Review by naree-hmrc — APPROVED** [review]
LGTM

File: app/models/BusinessType.scala (hunk 1 above)
Comment (AashuChauhanHmrc) [reply app/models/BusinessType.scala:59]: sure , we can look at it sometime.

**Review by naree-hmrc — APPROVED** [review]
LGTM

**Review by HarryHamilton — APPROVED** [review]

**Review by naree-hmrc — APPROVED** [review]

**Pull request merged**